.gitignore

```diff
@@ -1,2 +1,5 @@
+# IDE files
```

Comment:
It's usually better to ignore IDE-related files in a personal global `.gitignore` file - https://robots.thoughtbot.com/global-gitignore .

Could you please revert this change? :)

Reply:
Yes, i can do this. However many project use internal gitignore, like [servo](https://github.com/servo/servo/blob/master/.gitignore#L27). 
How do you do it? 
a. revert -> fix -> commit -> push
b. fix -> commit -> push 
or no difference?

Reply:
> However many project use internal gitignore, like servo.

Yep, not everybody agrees on this :(

> How do you do it?

It's small fix, so whatever you're comfortable with :)

Reply:
Fixed